plugins/oaiMetadataFormats/rfc1807/OAIMetadataFormat_RFC1807.inc.php

```diff
@@ -76,7 +80,7 @@ function toXml($record, $format = null) {
 			$this->formatElement('author', $creators) .
 			($article->getDatePublished()?$this->formatElement('date', $article->getDatePublished()):'') .
 			$this->formatElement('copyright', strip_tags($journal->getLocalizedSetting('copyrightNotice'))) .
-			($journal->getSetting('publishingMode') != PUBLISHING_MODE_NONE?$this->formatElement('other_access', "url:$url"):'') .
+			(($journal->getSetting('publishingMode') != PUBLISHING_MODE_NONE || $includeUrls)?$this->formatElement('other_access', "url:$url"):'') .
```

Comment:
I think this, `$journal->getSetting('publishingMode') != PUBLISHING_MODE_NONE`, is redundant because it's part of `$includeUrls`?